include/libopencm3/stm32/l0/timer.h

```diff
@@ -182,30 +218,52 @@ LGPL License Terms @ref lgpl_license
 /* 5 is reserved */
 #define LPTIM_CFGR_TRIGSEL_EXT_TRIG6	(6 << LPTIM_CFGR_TRIGSEL_SHIFT)
 #define LPTIM_CFGR_TRIGSEL_EXT_TRIG7	(7 << LPTIM_CFGR_TRIGSEL_SHIFT)
+/**@}*/
 
 #define LPTIM_CFGR_TRIGEN_SHIFT		17
+#define LPTIM_CFGR_TRIGSEL_MASK		0x07
 #define LPTIM_CFGR_TRIGEN		(3 << LPTIM_CFGR_TRIGEN_SHIFT)
+/* @defgroup LPTIM_CFGR_TRIGEN LPTIM_CFGR TRIGEN Trigger enable and polarity
```

Comment:
hah, this is what was giving me troubles with groupings, it's missing the second *  I've fixed it locally, no need to repush anything.